### PR TITLE
Add approvers

### PR DESCRIPTION
**Pull request by Jaanki**
Added Tom as approvers. Rearranged reviewers and approvers alphabetically.

Signed-off-by: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,17 +1,19 @@
 reviewers:
-- skeeey
-- zhiweiyin318
+- Jaanki
 - qiujian16
+- skeeey
 - skitt
+- tpantelis
+- zhiweiyin318
```
Comment (tpantelis) [inline OWNERS:7]: Why were all these lines shuffled around?

File: OWNERS [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 - dfarrell07
-- Jaanki
 - sridhargaddam
-- tpantelis
 - vthapar
 
 approvers:
-- zhiweiyin318
-- skeeey
+- Jaanki
```
Comment (tpantelis) [inline OWNERS:14]: I thought we had said we would add approvers according to the admins/owners in the submariner repos, ie skitt, me and mangelajo (tho he's not contributing in this repo right right now).

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (Jaanki) [reply OWNERS:7]: Rearranged alphabetically

Comment (dfarrell07) [reply OWNERS:7]: Are they alphabetical? The rending GitHub is showing me is [j,q,s,t,z,a,d,s,v].

Comment (Jaanki) [reply OWNERS:7]: This is split like this - all approvers first alphabetically and then all reviewers. The original setup was all approvers (non alphabetically) and then all reviewers (alphabetically)

File: OWNERS [hunk 3]
```diff
@@ -1,17 +1,18 @@
 reviewers:
-- skeeey
-- zhiweiyin318
 - qiujian16
+- skeeey
 - skitt
+- tpantelis
+- zhiweiyin318
```
Comment (Jaanki) [inline OWNERS:6]: All approvers first (rearranged alphabetically)

File: OWNERS [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 - aswinsuryan
 - dfarrell07
 - Jaanki
 - sridhargaddam
-- tpantelis
 - vthapar
```
Comment (Jaanki) [inline OWNERS:11]: All reviewers - were originally alphabetically ordered.

File: OWNERS [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
 
 approvers:
-- zhiweiyin318
-- skeeey
 - qiujian16
+- skeeey
 - skitt
+- tpantelis
+- zhiweiyin318
```
Comment (Jaanki) [inline OWNERS:18]: Rearranged approvers alphabetically

File: OWNERS (hunk 3 above)
Comment (tpantelis) [reply OWNERS:6]: I'd think the whole list would be arranged alphabetically rather than two sub-lists arranged separately.

File: OWNERS (hunk 1 above)
Comment (skitt) [reply OWNERS]: > The original setup was all approvers (non alphabetically) and then all reviewers (alphabetically)

That was only the result of adding people to the reviewers list and not the approvers list, there’s no underlying intent.

**Review by skitt — APPROVED** [review]

**Pull request merged**